src/theory/bv/rewrites

```diff
@@ -6,204 +6,327 @@
 ; extracts the range [i..j]. Vide https://smtlib.cs.uiowa.edu/theories-FixedSizeBitVectors.shtml
```

Comment:
These comments are a bit without context. What extracts the range? The extract command but what are i and j? Why are these mentioned here?